open-sphere-base/analysis/src/main/java/io/opensphere/analysis/baseball/BaseballPanel.java

```diff
@@ -0,0 +1,377 @@
+     * @return the String representation of the value
+     */
+    private String getValueAsString(String key, Object value, DataElement dataElement)
+    {
+        String returnValue;
+        if (value == null)
+        {
+            returnValue = null;
+        }
+        else if (value instanceof Double)
+        {
+            SpecialKey specialType = dataElement.getDataTypeInfo().getMetaDataInfo().getSpecialTypeForKey(key);
+            if (specialType instanceof LatitudeKey)
+            {
+                returnValue = BaseballUtils.formatCoordinate((Double)value, myPositionFormat, CoordType.LAT);
+            }
+            else if (specialType instanceof LongitudeKey)
+            {
+                returnValue = BaseballUtils.formatCoordinate((Double)value, myPositionFormat, CoordType.LON);
+            }
+            else
+            {
+                returnValue = BaseballUtils.formatNumber((Number)value);
+            }
+        }
+        else if (value instanceof Number)
+        {
+            returnValue = BaseballUtils.formatNumber((Number)value);
+        }
+        else if (value instanceof Date)
+        {
+            returnValue = BaseballUtils.formatDate((Date)value, myToolbox.getPreferencesRegistry().getPreferences(ListToolPreferences.class)
+                    .getInt(ListToolPreferences.LIST_TOOL_TIME_PRECISION_DIGITS, 0));
+        }
+        else if (value instanceof TimeSpan)
+        {
+            returnValue = BaseballUtils.formatTimeSpan((TimeSpan)value, myToolbox.getPreferencesRegistry()
+                    .getPreferences(ListToolPreferences.class).getInt(ListToolPreferences.LIST_TOOL_TIME_PRECISION_DIGITS, 0));
+        }
+        else
+        {
+            returnValue = value.toString();
+        }
+        return returnValue;
+    }
+
+    /**
+     * Clears the styles for the coordinate buttons, then sets the style for selected button.
+     *
+     * @param button the button
+     */
+    private void setButtonStyles(Button button)
+    {
+        myDDButton.setStyle("");
+        myDMSButton.setStyle("");
+        myDDMButton.setStyle("");
+        myMGRSButton.setStyle("");
+        button.setStyle(BUTTON_STYLE);
+    }
+
+    /**
+     * Sets the coordinate label based off the currently active data element
+     * and the selected coordinate format.
+     */
+    private void setCoordinates()
+    {
+        if (myActiveDataElement == null)
+        {
+            myCoordinates.setText("");
+        }
+        else if (myPositionFormat == GeographicPositionFormat.MGRS)
+        {
+            myCoordinates.setText(BaseballUtils.formatMGRS(myActiveDataElement));
+        }
+        else
+        {
+            String latitudeKey = myActiveDataElement.getDataTypeInfo().getMetaDataInfo().getLatitudeKey();
+            Object latitudeValue = myActiveDataElement.getMetaData().getValue(latitudeKey);
+        	String latitude = getValueAsString(latitudeKey, latitudeValue, myActiveDataElement);
```

Comment:
indentation, don't use tabs
```suggestion
            String latitude = getValueAsString(latitudeKey, latitudeValue, myActiveDataElement);
```